synpress.config.js

```diff
@@ -38,7 +33,7 @@ module.exports = defineConfig({
     testIsolation: false,
     setupNodeEvents,
     baseUrl: 'http://localhost:3000',
-    specPattern: specPattern[process.env.EXTENSION],
+    specPattern: 'tests/e2e/specs/**/*spec.{js,jsx,ts,tsx}',
     supportFile,
```

Comment:
Just to confirm, won't it be `process.env.CYPRESS_SPEC_PATTERN`?

Reply:
no.
but the user can provide an env variable, which will override this setting

Reply:
Make sense. But where we are doing it?